packages/cli/src/index.ts

```diff
@@ -330,6 +335,11 @@ ${styles.bold('Examples:')}
     )
     .option('--target <name>', 'target to use in profiles.yml file', undefined)
     .option('-y, --assume-yes', 'assume yes to prompts', false)
+    .option(
+        '--exclude-dimension-types',
```

Comment:
have we considered the option  `--exclude-meta-property` so it has a broader coverage ? or do we want the option to be really specific ?

Reply:
oh that's good suggestion!